src/Policies/Policy.php

```diff
@@ -84,10 +84,10 @@ public function enforce(): self
         return $this;
     }
 
-    public function reportTo(string $uri): self
+    public function reportTo(string $reportTo, string $reportToUri = ''): self
     {
-        $this->directives[Directive::REPORT] = [$uri];
-        $this->directives[Directive::REPORT_TO] = [$uri];
+        $this->directives[Directive::REPORT] = [$reportToUri ?? $reportTo];
```

Comment:
This operator only checks if the variable is `null` so it will still return the empty string if the env variable is not set. Perhaps, you could use a ternary instead? Also, you could remove the default argument value `$reportToUri = '' `, the `getEnv()` should return a falsy value to invalidate this in case it's not set.
```suggestion
        $this->directives[Directive::REPORT] = [$reportToUri ?: $reportTo];
```

Reply:
The default `$reportToUri = ''` is so that it doesn't break the existing unit test that passes only a single parameter 🤔